src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -774,7 +774,12 @@ async Task Consume(bool async)
         /// <summary>
         /// Closes the <see cref="NpgsqlDataReader"/> reader, allowing a new command to be executed.
         /// </summary>
-        public Task CloseAsync() => Close(false, true);
+#if !NET461 && !NETSTANDARD2_0
```

Comment:
Thanks. These new ADO.NET async APIS should actually be available in .NET Standard 2.1 already, so we should change these #if conditions.

But let's do that in a different PR, this one is already big enough :)